### DISCUSSION TITLE
Cannot use colors from extended color palette

### DISCUSSION THREAD

**Discussion by MartinMa** [tailwindlabs/tailwindcss#5104] [Help] [answered]
I'm trying to use the `fill-amber-300` utility class to change the fill color of an embedded svg. The fill style is not applied. On the other hand, `fill-yellow-300` works. I tried to override the amber class using the config which didn't work either. What could possibly go wrong here?

I tried with and without JIT mode.

See play link for reproduction:

https://play.tailwindcss.com/7rSPaqBWD7?file=config

![image](https://user-images.githubusercontent.com/485747/127275281-c17da79b-cd4e-4502-8285-eadff82fc454.png)

**Comment by simonswiss** [comment, marked as answer]
Hey!

In your Tailwind Play, you're trying to access the colors with the `theme()` function, which doesn't use the `colors` that you have imported at the top of your config file, but the theme's existing colors instead.

And... `amber` does not exist in the default theme colors.

To use the colors you have imported,  you need to use the `colors` object directly:

```diff
const colors = require('tailwindcss/colors')

module.exports = {
  theme: {
    fill: (theme) => ({
-      'amber': theme('colors.amber.300'),
-      'yellow': theme('colors.yellow.300'),
+      amber: colors.amber[300],
+      yellow: colors.yellow[300],
    }),
  },
  variants: {},
  plugins: [],
}

```

Here's your updated Tailwind Play with the correct way to pull in the colors imported from `tailwindcss/colors`:

https://play.tailwindcss.com/zjiiIeFySX?file=config

![image](https://user-images.githubusercontent.com/485747/127274591-37d738a6-27b5-4fce-8d50-1c7d64f7cc2a.png)

Moving this to our [discussions](https://github.com/tailwindlabs/tailwindcss/discussions), since this might help someone else 👍

---

Semi-relevant side note, and  "fun" fact to add to the confusion. The default `yellow` is the default theme is actually... `amber` 😅 

So when you were doing `yellow: theme('colors.yellow.300')`, you were actually getting the `amber-300` color!

**Reply by MartinMa** [reply]
Interesting. Thanks for the answer and explanation.

I think the documentation on https://tailwindcss.com/docs/customizing-colors is misleading. It states at the top that tailwind has a default color palette included out-of-the-box and under Yellow the key `colors.amber` is listed, when it actually is `colors.yellow`.

![Bildschirmfoto 2021-07-28 um 08 41 21](https://user-images.githubusercontent.com/123297/127276492-c1efef26-63f3-4017-95a6-5cb8e7db9481.png)

Further down in the Curating colors paragraph, the concept of aliasing is explained and how it affects the default theme. But I think it is confusing if you just look at the first table for colors, want to use them and then the listed keys fail to work.

**Reply by simonswiss** [reply]
Thanks for the feedback!

I can see the confusion. What it's _trying_ to say is that on the default color palette, the default `yellow` is actually the `amber` colors from the complete palette.

So, default yellow _is_ indeed `colors.amber`. You can see it in [the default config file](https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L23):

![image](https://user-images.githubusercontent.com/485747/127416617-e2093225-723b-4e31-a2f8-6ec30a6d478d.png)

Similarly, the default `gray` is the `coolGray`, the default `green` is actually `emerald`.

## Why would we do such a thing? 

We wanted the default colors to be named after real colors, instead of "conceptual names" like `emerald`, `amber`, etc.

Most non-native English speakers will likely recognise green and yellow better than emerald and amber.

And I bet a lot of native English speakers do not know what color an emerald stone is, as well 👍
